README.md

```diff
@@ -1,3 +1,7 @@
-Theia will contain Network Flow Visibility functions which are split from the main repo. Flow exporter and flow aggregator will be kept in main repo, other modules, such as Clickhouse and Grafana related parts, will be moved to this new repo.
+# Theia
 
-During the code migration period, we will still keep related functions available and stable in main repo, but the functions in this new repo won't be ready unless we announce the migiration is completed. 
+Theia will contain Network Flow Visibility functions which are split from the main repo. Flow
+exporter and flow aggregator will be kept in main repo, other modules, such as Clickhouse and Grafana related parts, will be moved to this new repo.
```

Comment:
nit: We all know "antrea" is the main repo, but not every user might know that. Consider referring to antrea-io/antrea repo